common/controllers/collection/render-as-table.test.js

```diff
@@ -0,0 +1,122 @@
+const controller = require('./render-as-table')
```

Comment:
Is this the first test for `render-as-table`? :scream: :heart:

Reply:
Yep... :/